### PR TITLE
Drupal, Hippies and backend restrictions

### PR DESCRIPTION
**Pull request by RoySegall**
@amitaibu ^^

### CODE REVIEW

**Comment by RoySegall** [comment]
Could not think of a better title for the blog post, Sorry.

**Comment by amitaibu** [comment]
Marking this for myself for review.

File: app/_posts/2015-12-15-apps-entity-restrictions.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - "Drupal-planet"
+permalink: "/content/apps-entity-restrictions"
+author: RoySegall
+layout: post
+published: true
+---
+
+{% include setup %}
+
+Have you ever heard on the Burning man project? No? Three months earlier neither
```
Comment (bricel) [inline app/_posts/2015-12-15-apps-entity-restrictions.md]: heard of ?

File: app/_posts/2015-12-15-apps-entity-restrictions.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+am I. I can describe it in one simple sentence: a lot of Hippies settle down in
```
Comment (bricel) [inline app/_posts/2015-12-15-apps-entity-restrictions.md]: am I => did I
settle => settled
2

File: app/_posts/2015-12-15-apps-entity-restrictions.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+the desert for couple of weeks and set up small camps as part of a big city when
+each camp have it’s own theme.
+
+And here I am going to the hackathon of the local Burning man community as an
```
Comment (bricel) [inline app/_posts/2015-12-15-apps-entity-restrictions.md]: And here I am going to the => And here I find myself going to the

File: app/_posts/2015-12-15-apps-entity-restrictions.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+enthusiastic Drupal developer willing to solve any Drupal issue. My part was to
+write the backend part. The obvious choice was the Restful module. But then I
```
Comment (bricel) [inline app/_posts/2015-12-15-apps-entity-restrictions.md]: backend part. => backend.

File: app/_posts/2015-12-15-apps-entity-restrictions.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+came across a big problem: How can I manage 3rd party applications and make sure
+they can’t access resources which they should’nt access? How can I prevent from
```
Comment (bricel) [inline app/_posts/2015-12-15-apps-entity-restrictions.md]: should’nt =>  shouldn't

**Comment by RoySegall** [comment]
Ready for another review.

**Comment by amitaibu** [comment]
closed in favor of #274

**Pull request closed without merging**